kodilla-rps/src/main/java/com/kodilla/rps/User.java

```diff
@@ -1,12 +1,16 @@
 package com.kodilla.rps;
 
 public class User {
-    private String name;
-    private int rounds;
+    private final String name;
+    private final int rounds;
+    private int joker;
+    private int difficulty;
```

Comment:
to samo co [tu](https://github.com/robert-er/kodilla-course-m6/pull/4/commits/9a605f2239a526119da95ddbf7dba1c4f2d9453a#r443719936)